sway/tree/container.c

```diff
@@ -236,6 +235,45 @@ struct sway_container *container_output_destroy(struct sway_container *output) {
 	return &root_container;
 }
 
+struct sway_container *container_workspace_destroy(
```

Comment:
probably belongs in workspace.c

Reply:
We already have `container_output_destroy` and `container_view_destroy` in this file, I'll move them all